raster/r.neighbors/main.c

```diff
@@ -75,6 +81,15 @@ static struct menu menu[] = {
     {0, 0, 0, 0, 0, 0, 0, 0}
 };
 
+/* modify this table to add new methods */
+static struct weight_functions weight_functions[] = {
+    {"None", "No weighting"},
```

Comment:
In parameters in GRASS GIS, the none value is usually all-lowercase `none` and not Python-like `None`. We can consider generally supporting an alias which is seemingly correct from Python perspective, but `none` is currently prevalent at least in display modules, so it should be the primary choice here.

Reply:
OK, all occurrences of `none` are now changed to all-lowercase.